tests/unit/src/main/java/org/mule/tck/util/MuleContextUtils.java

```diff
@@ -22,6 +22,7 @@
 import static org.mule.runtime.config.internal.error.MuleCoreErrorTypeRepository.MULE_CORE_ERROR_TYPE_REPOSITORY;
 import static org.mule.runtime.core.api.config.MuleManifest.getProductVersion;
 import static org.mule.runtime.core.api.config.MuleProperties.OBJECT_STORE_MANAGER;
+import static org.mule.runtime.core.api.config.bootstrap.ArtifactType.APP;
```

Comment:
?

Reply:
Needed so that the muleContext is mocked with an ArtifactType,

Reply:
Added a missing line of code. :(